src/cloud/paths/provision_user_delete.yml

```diff
@@ -0,0 +1,30 @@
+post:
+  operationId: PostProvisionUserDelete
+  tags:
+    - ProvisionUserDelete
+  description: Delete a user from an organization. Deletes the user if they are part of no other organization. Idempotent.
+  requestBody:
+    description: User ID to delete
```

Comment:
The description tripped me up a little as it's not quite accurate. Maybe something like "The request includes the User ID and the Org ID to delete the user and whether to transfer user resources to an optional new owner"